Core.UnitTests/Caching/TypeCacheTests/GetOrCreateAdditionalType_TypeCacheTest.cs

```diff
@@ -70,32 +71,33 @@ public void CacheMiss_UsesAssemblyContextFromPool ()
 
       bool isDequeued = false;
       _assemblyContextPoolMock
-          .Expect (mock => mock.Dequeue())
-          .Return (assemblyContext)
-          .WhenCalled (mi => { isDequeued = true; });
+          .Setup (mock => mock.Dequeue())
+          .Returns (assemblyContext)
+          .Callback (() => { isDequeued = true; })
+          .Verifiable();
 
       _typeAssemblerMock
-          .Expect (
-              mock => mock.AssembleAdditionalType (
-                  additionalTypeID,
-                  assemblyContext.ParticipantState,
-                  assemblyContext.MutableTypeBatchCodeGenerator))
-          .Return (new TypeAssemblyResult (additionalType))
-          .WhenCalled (mi => Assert.That (isDequeued, Is.True));
+          .Setup (mock => mock.AssembleAdditionalType (
+              additionalTypeID,
+              assemblyContext.ParticipantState,
+              assemblyContext.MutableTypeBatchCodeGenerator))
+          .Returns (new TypeAssemblyResult (additionalType))
+          .Callback (() => Assert.That (isDequeued, Is.True))
+          .Verifiable();
 
       _assemblyContextPoolMock
-          .Expect (mock => mock.Enqueue (assemblyContext))
-          .WhenCalled (
-              mi =>
-              {
-                Assert.That (isDequeued, Is.True);
-                isDequeued = false;
-              });
+          .Setup (mock => mock.Enqueue (assemblyContext))
+          .Callback (() =>
```

Comment:
new line.  this is "everywhere" from the looks of it. I'll be stopping the review for a bit until this is fixed so I don't get the redone files.

Reply:
What I did was I reformatted the whole UnitTest project and reset the changes that weren't "that". Hopefully it fixed everything without introducing new formatting errors